src/pages/ConnectPayment/sections/ConnectPayment.tsx

```diff
@@ -103,123 +101,33 @@ const ConnectPaymentImage = styled('img')({
   },
 })
 
-interface State {
-  trustlyModalIsOpen: boolean
-  trustlyUrl: string | null
+export const ConnectTrustlyPage: React.FC = () => {
+  const textKeys = useTextKeys()
+  const market = useMarket()
+
+  return (
+    <>
+      <Background />
+      <InnerWrapper>
+        <TextColumn>
+          <Header>
+            <HeaderPart>
+              {textKeys.ONBOARDING_CONNECT_DD_PRE_HEADLINE()}
+            </HeaderPart>
+            <HeaderPart>{textKeys.ONBOARDING_CONNECT_DD_HEADLINE()}</HeaderPart>
+          </Header>
+          <ConnectText>{textKeys.ONBOARDING_CONNECT_DD_BODY()}</ConnectText>
```

Comment:
Could be argued this should have a different text key for adyen since it is not Direct Debit

Reply:
True. But i think lets focus on copy later if it comes up, its easy to detect and easy to change :)